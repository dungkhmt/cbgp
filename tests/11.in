30 30
30 43
17 12
12 22
22 24
24 9
9 4
4 23
23 20
20 17
2 28
28 14
14 8
8 15
15 3
3 19
19 5
5 30
30 2
27 13
13 25
25 6
6 27
11 16
16 26
26 7
7 18
18 11
10 21
21 29
29 1
1 10
28 12
27 2
8 4
7 1
19 20
22 9
30 17
10 11
25 15
14 13
5 6
23 3
29 16
1
15 3 21 11 7 1 0 0 30 2 12 2 27 3 7 3 16 16 17 5 19 7 19 15 12 4 10 6 8 2 23 4 30 4 21 9 31 1 34 0 19 5 18 16 6 0 17 17 10 2 25 4 13 3 18 14 20 4 29 3 
