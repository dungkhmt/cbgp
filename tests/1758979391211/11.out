(16, 11), (11, 21), (21, 23), (23, 8), (8, 3), (3, 22), (22, 19), (19, 16), (1, 27), (27, 13), (13, 7), (7, 14), (14, 2), (2, 18), (18, 4), (4, 29), (29, 1), (26, 12), (12, 24), (24, 5), (5, 26), (10, 15), (15, 25), (25, 6), (6, 17), (17, 10), (9, 20), (20, 28), (28, 0), (0, 9), (27, 11), (26, 1), (7, 3), (6, 0), (18, 19), (21, 8), (29, 16), (9, 10), (24, 14), (13, 12), (4, 5), (22, 2), (28, 15), 
numberIntersection=0 
Var(nodePos)=6741.961111111111 
minAngle=0.008403163547646919 
minDistance=0.008403064652344154 
Var(edgeLength)=2514.4530278188336 
sumAngle=36.123061869381715 
sumDistance=103200.85452292151 
minEdgeLength=1.0 
sumEdgeLength=2395.611697719793 

0.0000000000 6741.9611111111 0.0084031635 0.0084030647 2514.4530278188 36.1230618694 103200.8545229215 1.0000000000 2395.6116977198

0: (117, 3),
1: (2, 118),
2: (118, 1),
3: (0, 1),
4: (119, 2),
5: (3, 3),
6: (118, 3),
7: (1, 1),
8: (0, 120),
9: (3, 116),
10: (117, 117),
11: (2, 119),
12: (2, 2),
13: (1, 2),
14: (118, 2),
15: (117, 116),
16: (120, 119),
17: (118, 117),
18: (119, 1),
19: (120, 0),
20: (4, 116),
21: (1, 119),
22: (0, 0),
23: (120, 120),
24: (3, 2),
25: (117, 4),
26: (2, 3),
27: (1, 118),
28: (116, 116),
29: (119, 118),

Time taken: 47.310 s
117 3 2 118 118 1 0 1 119 2 3 3 118 3 1 1 0 120 3 116 117 117 2 119 2 2 1 2 118 2 117 116 120 119 118 117 119 1 120 0 4 116 1 119 0 0 120 120 3 2 117 4 2 3 1 118 116 116 119 118 
0: (117, 3), 1: (2, 118), 2: (118, 1), 3: (0, 1), 4: (119, 2), 5: (3, 3), 6: (118, 3), 7: (1, 1), 8: (0, 120), 9: (3, 116), 10: (117, 117), 11: (2, 119), 12: (2, 2), 13: (1, 2), 14: (118, 2), 15: (117, 116), 16: (120, 119), 17: (118, 117), 18: (119, 1), 19: (120, 0), 20: (4, 116), 21: (1, 119), 22: (0, 0), 23: (120, 120), 24: (3, 2), 25: (117, 4), 26: (2, 3), 27: (1, 118), 28: (116, 116), 29: (119, 118), 
numberIntersection=0 minAngle=0.008403163547646919 sumAngle=36.12306186937091 minEdgeLength=1.0 sumEdgeLength=2395.6116977197917 minDistance=0.008403064652344154 sumDistance=103200.85452292222 
